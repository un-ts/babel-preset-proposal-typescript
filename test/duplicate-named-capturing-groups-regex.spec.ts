import _ from './duplicate-named-capturing-groups-regex.js'
import { execute } from './helpers.js'

const proposal = 'duplicate-named-capturing-groups-regex'

test(proposal, () => {
  expect(() => execute(proposal)).toThrowErrorMatchingSnapshot()
  expect(_('02-1999')).toMatchInlineSnapshot(`"1999"`)
})
