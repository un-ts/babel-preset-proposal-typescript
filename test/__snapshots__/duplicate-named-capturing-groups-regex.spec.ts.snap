// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`duplicate-named-capturing-groups-regex 1`] = `"Invalid regular expression: /(?<year>\\d{4})-(?<month>\\d{2})|(?<month>\\d{2})-(?<year>\\d{4})/: Duplicate capture group name"`;
